templates/ssm-parameters.j2

```diff
@@ -0,0 +1,46 @@
+# This is a cloudformation jinjaized template to install parameters into the
+# SSM parameter store.  This is only for non-securestring parameters.
+# Cloudformation does not support adding securestring parameters into the SSM.
+#
+# Usage:
+#  Create a sceptre template `my-ssm-params.yaml`
+#
+#  template_path: remote/ssm-parameters.j2
+#  hooks:
+#    before_launch:
+#      - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/master/ssm-parameters.j2 --create-dirs -o templates/remote/ssm-parameters.j2"
+#  stack_name: my-ssm-vars
+#  sceptre_user_data:
+#    Prefix: /my-ssm-vars/
+#    Parameters:
+#      - Name: param1
+#        Value: foo
+#      - Name: param2
+#        Value: foo
+
+Description: "Setup parameters in the SSM parameter store"
+AWSTemplateFormatVersion: "2010-09-09"
+Resources:
+  {% for parameter in sceptre_user_data.Parameters %}
```

Comment:
I don't have an easy way to test this, but it looks like this might create an extra newline where you don't want it. Would it make sense to change either the `{%` to `{%-`, or `%}` to `-%}` ?

Minor issue either way I guess

Reply:
Why would you think this would create an extra newline?  The purpose of this PR is to move this template, which is already working in production, to a new location.  take a look at https://github.com/Sage-Bionetworks/synapse-login-aws-infra/pull/42